interfaces/repo_test.go

```diff
@@ -910,3 +939,28 @@ func (s *RepositorySuite) TestSecuritySnippetsForSnapFailure(c *C) {
 	c.Assert(err, ErrorMatches, "cannot compute snippet for consumer")
 	c.Check(snippets, IsNil)
 }
+
+func (s *RepositorySuite) TestSecuritySnippetsForSnapFailureWithStaticSnippets(c *C) {
+	var testSecurity SecuritySystem = "security"
+	iface := &TestInterface{
+		InterfaceName: "interface",
+		StaticSlotSecuritySnippetCallback: func(slot *Slot, securitySystem SecuritySystem) ([]byte, error) {
+			return nil, fmt.Errorf("cannot compute static snippet for consumer")
+		},
+		StaticPlugSecuritySnippetCallback: func(plug *Plug, securitySystem SecuritySystem) ([]byte, error) {
+			return nil, fmt.Errorf("cannot compute static snippet for provider")
```

Comment:
The language of 'consumer' and 'provider' here seems at odds with the changes above where you renamed consumer to slot and producer to plug. Should this be more consistent?